sdk/storage/azure-storage-blob/src/test/java/com/azure/android/storage/blob/BlobTest.java

```diff
@@ -929,4 +1042,30 @@ public void onFailure(Throwable throwable, Response response) {
     }
 
     // setTagsError tested in AC fail as it throws BlobStorageException
+
+    @Test
+    public void getTagsAsync() {
+        // Setup
+        CountDownLatch latch = new CountDownLatch(1);
+
+        // When
+        asyncClient.getBlobTags(containerName, blobName, new CallbackWithHeader<Map<String, String>, BlobGetTagsHeaders>() {
+                @Override
+                public void onSuccess(Map<String, String> result, BlobGetTagsHeaders header, Response response) {
+                    assertEquals(200, response.code());
```

Comment:
Should we check that the result's contents are correct like we check the body contents in `rawRownloadAsync()`?